Fraudio/index.ts

```diff
@@ -0,0 +1,48 @@
+import * as selectively from "selectively"
+import * as authly from "authly"
+
+export interface Fraudio {
+	score: number
+	recommendation: "green" | "yellow" | "red"
+	id: authly.Identifier
```

Comment:
string, never assume externally provided id:s are dividable by 4 (which authly.Identifier always are)
```suggestion
	id: string
```